pipelines/broad/arrays/validate_chip/ValidateChip.wdl

```diff
@@ -65,7 +65,7 @@ workflow ValidateChip {
   call InternalTasks.CreateExtendedIlluminaManifest {
     input:
       input_csv = chip_manifest_csv_file,
-      output_base_name = chip_type + ".1.4",
+      output_base_name = chip_type + ".1.5",
```

Comment:
maybe this should be a workflow input?

Reply:
Yeah.  I kind of would like the picard tool to name the output file with the version number (since it is a static in the tool).